tests/eulertest.cpp

```diff
@@ -118,3 +118,21 @@ TEST_F(EulerTest, MultipleSame) {
 	EXPECT_CLOSE(e2.GetNextNetworkState()["z"], 8);
 	EXPECT_CLOSE(e.GetNextNetworkState()["z"], 8);
 }
+
+TEST_F(EulerTest, KahanSummationEqual) {
```

Comment:
Kahan Summering bruges til at undgå en akkumuleret fejl når man tager summen af en masse floating point numbers. Når man kun har to, så gør det jo ikke super meget. Ideelt burde en unit test fejle hvis du erstatter KahanSummation imlpmentationen med et naivt summationsalgoritme, men det gør den ikke.

Den naive summation jeg brugte, for reference
```cpp
double EulerEvaluator::KahanSummation(std::initializer_list<double> numbers) {
	double sum = 0.0;
	for (const auto &number : numbers) {
		sum += number;
	}
	return sum;
}
```